examples/universal-cnf/vppagent/pkg/config/config_endpoints.go

```diff
@@ -62,7 +68,12 @@ func NewProcessEndpoints(backend UniversalCNFBackend, endpoints []*nseconfig.End
 			IPAddress:              e.VL3.IPAM.DefaultPrefixPool,
 			Routes:                 nil,
 		}
-
+		var err error
+		service := newService(e.VL3.IPAM.ServerAddress)
```

Comment:
`service` is to broad of a name, this is a particular service, not any service.

Reply:
done